renku_notebooks/api/schemas/servers_post.py

```diff
@@ -51,6 +56,14 @@ class LaunchNotebookRequestWithStorage(LaunchNotebookRequestWithoutStorage):
     )
 
 
+class Renku1LaunchNotebookRequest(LaunchNotebookRequestBase):
+    """To validate start request for Renku 1 sessions."""
+
+    # TODO: Make Renku1 a feature flag and add these fields to a specific Renku1 class
+    renku_1_project_id = fields.String(required=True)
+    renku_1_environment_id = fields.String(required=True)
```

Comment:
This should be the ID of a session launcher:
```suggestion
    renku_1_session_launcher_id = fields.String(required=True)
```